include/odp/api/spec/packet_io_stats.h

```diff
@@ -26,42 +26,23 @@ extern "C" {
  */
 
 /**
- * Packet IO statistics
- *
- * Packet IO statistics counters follow RFCs for Management Information Base
- * (MIB)for use with network management protocols in the Internet community:
- * https://tools.ietf.org/html/rfc3635
- * https://tools.ietf.org/html/rfc2863
- * https://tools.ietf.org/html/rfc2819
+ * Packet IO statistics counters
  */
 typedef struct odp_pktio_stats_t {
-	/**
-	 * The number of octets in valid MAC frames received on this interface,
-	 * including the MAC header and FCS. See ifHCInOctets counter
-	 * description in RFC 3635 for details.
-	 */
+	/** Number of octets in successfully received packets (includes MAC
+	 *  header and FCS). */
 	uint64_t in_octets;
```

Comment:
From my experience, sometimes control (pause) packets may distort picture of packet rate observed by the hardware, due to they take certain bandwidth. I assume, they are not calculated in the total number of in_packets/in_octets?

Reply:
Update in V4